### DISCUSSION TITLE
Self-Host Guideline

### DISCUSSION THREAD

**Discussion by wahmedswl** [estuary/flow#1325] [General]
Hi!

Is there any guide to self-host the Estuary Flow? Something like docker compose file or anything? There seems to be a Tilt file so is that's way to move forward?

Thanks

**Comment by dyaffe** [comment]
Hi Waqas,

Thanks for reaching out -- currently it's not simple to self host.  We are
working on making it easier, but Estuary Flow is a big distributed system,
so doing so takes a bit of work.  If you are interested we could try to
help you, but the simplest way to get started is definitely through using
the SaaS product.

Thanks,
Dave

On Mon, Dec 25, 2023 at 1:26 AM Waqas Ahmed ***@***.***>
wrote:

> Hi!
>
> Is there any guide to self-host the Estuary Flow? Something like docker
> compose file or anything? There seems to be a Tilt file so is that's way to
> move forward?
>
> Thanks
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/estuary/flow/discussions/1325>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB5YPZMVEMTV4FCNX7NTKETYLEMATAVCNFSM6AAAAABBCCWUXCVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVHE4TMNRYGI>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 

David Yaffe

Co-Founder



e: ***@***.***

www.estuary.dev
